smithy-linters/src/test/resources/software/amazon/smithy/linters/errorfiles/deprecated-traits-validator.errors

```diff
@@ -0,0 +1 @@
+[WARNING] ns.foo#myTraitName: The myTraitName trait is deprecated. | Deprecated Traits
```

Comment:
This should be applied to the shape that used the deprecated trait, not the deprecated trait itself. When I see this warning, I need to know what shape the trait was applied to and what trait was applied.

Reply:
Fixed in 8abdbc17